cli/wallet/multisig.go

```diff
@@ -84,10 +87,19 @@ func signStoredTransaction(ctx *cli.Context) error {
 		if err != nil {
 			return cli.NewExitError(fmt.Errorf("failed to submit transaction to RPC node: %w", err), 1)
 		}
-		fmt.Fprintln(ctx.App.Writer, res.StringLE())
-		return nil
+		resTx = tx.Hash().StringLE()
+		if ctx.Bool("await") {
+			version, err := c.GetVersion()
+			if err != nil {
+				return cli.NewExitError(fmt.Errorf("failed to get version from RPC node: %w", err), 1)
+			}
+			_, err = actor.NewWaiter(c, version).Wait(tx.Hash(), tx.ValidUntilBlock, nil)
+			if err != nil {
+				return cli.NewExitError(fmt.Errorf("the transaction doesn't accepted to the block: %w", err), 1)
+			}
+			resTx = res.StringLE()
+		}
 	}
-
-	fmt.Fprintln(ctx.App.Writer, tx.Hash().StringLE())
+	txctx.FmtPrintTransaction(ctx.App.Writer, resTx, ctx.Bool("await"))
```

Comment:
Provide resulting `state.AppExecResult` for dump here and everywhere where `Wait` is used for tx awaiting.